hypershift-operator/controllers/nodepool/inplace.go

```diff
@@ -44,6 +45,23 @@ func (r *NodePoolReconciler) reconcileMachineSet(ctx context.Context,
 		return err
 	}
 
+	// Set MaxUnavailable for the inplace upgrader to use
+	intOrPercent := intstr.FromInt(1)
+	if nodePool.Spec.Management.InPlace != nil {
```

Comment:
can we please have all this added logic into a unit tested func e.g `getMaxUnavailable`?

Reply:
Will do